homeassistant/helpers/entity.py

```diff
@@ -425,6 +428,20 @@ def _name_translation_key(self) -> str | None:
             f".{self.translation_key}.name"
         )
 
+    def _name_placeholders(self, name: str) -> str:
```

Comment:
This is not a good name for this function; the name suggests it returns placeholders when it in fact processes translations, including substituting translation placeholders

Reply:
Renamed to `_substitue_name_placeholders`